Source/WebKit/Shared/ImageOptions.h

```diff
@@ -30,6 +30,7 @@ namespace WebKit {
 
 enum ImageOptions {
     ImageOptionsShareable = 1 << 0,
+    ImageOptionsLocalPDFSnapshot = 1 << 1,
```

Comment:
This should be something more generic, like "Use local buffer".

Reply:
Fixed.